docs/examples/package.json

```diff
@@ -2,17 +2,26 @@
   "name": "@epfml/disco-node-example",
   "private": true,
   "description": "In `training.ts` we give a brief example of discojs, in it we run two clients training jointly via federated learning. It trains  on a few examples of the [face task](https://www.kaggle.com/datasets/frabbisw/facial-age), the samples are already stored in the  repo and so it is not necessary to download any additional data.",
-  "main": "training.ts",
   "scripts": {
     "train": "npm run build && node dist/training.js",
     "custom_task": "npm run build && node dist/custom_task.js",
     "build": "tsc",
-    "lint": "npx eslint --max-warnings 0 .",
-    "test": ": nothing"
+    "lint": "npx eslint --ext ts --max-warnings 0 .",
+    "test": "npm run train"
   },
   "license": "ISC",
   "dependencies": {
     "@epfml/disco-server": "*",
+    "@epfml/discojs-core": "*",
     "@epfml/discojs-node": "*"
   },
+  "devDependencies": {
```

Comment:
Should we add an instruction for installing dependencies in the docs/examples/README.md ? Is docs/examples included in the npm -ws commands?

Reply:
hum, good question; yes? I'm not really sure that I understand the audience for this example.
 * if it is for devs to have a starter project based on disco, I think linter, how to deps and such are out of scope IMO as they would uses their own coding pratices
   * in this case, it won't be inside `workspaces` as it needs to be very independant
 * if it is for devs **of** disco, to understand how it is used, yeah, linter, how to deps, … does make sense
   * but why not directing them to directly read tests? that way, they can directly jump in code, it also pushes us to better write theses

Reply:
I was thinking of these examples for external users so yea I don't think making it a workspace is a good idea